articles/organizations/configure-organizations.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Configure Organizations
+description: Learn to create organizations using Auth0's Organizations feature.
+topics:
+  - organizations
+contentType: 
+    - how-to
+    - concept
+useCase:
+  - build-an-app
+toc: true
+---
+
+# Configure Organizations
+
+To begin using the Organizations feature, you must create and configure your organizations.
+
+## Create organizations
```

Comment:
We're missing sections for all GET endpoints for Organizations, Organization Connections, Organization Members, Organization Member Roles and the other endpoint "Get A User's Organizations" - see: https://auth0team.atlassian.net/wiki/spaces/ORGS/pages/766412930/Organization+Support+in+the+Management+API+WIP